app/views/application_templates/show.haml

```diff
@@ -145,8 +145,15 @@
         = select_tag :gender,
           options_for_select(ApplicationSubmission::GENDER_OPTIONS),
           include_blank: 'Prefer not to answer', class: 'custom-select'
-      - unless @current_user.try(:staff?)
-        .form-group
-          = submit_tag 'Submit application', class: 'btn btn-primary'
+    - if @template.resume_upload_enabled?
+      %h2.heading Resume
+      .explanation
+        Please upload a copy of your resume.
+        - #PLACEHOLDER, IDK WHAT TO REALLY PUT
```

Comment:
Wording is fine. However, could we make the upload required? If someone has enabled resume upload the chances are slim that they'd want applications without resumes.